src/main/java/hyperledger/besu/java/rest/client/config/TaskExecutorConfig.java

```diff
@@ -0,0 +1,21 @@
+package hyperledger.besu.java.rest.client.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.task.TaskExecutor;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+@Configuration
+public class TaskExecutorConfig {
+
+  @Bean
+  public TaskExecutor threadPoolTaskExecutor() {
+
+    ThreadPoolTaskExecutor threadPoolTaskExecutor = new ThreadPoolTaskExecutor();
+    threadPoolTaskExecutor.setCorePoolSize(7);
+    threadPoolTaskExecutor.setMaxPoolSize(7);
```

Comment:
Let's make these parameters as configurable, let's not hardcode it. We can have these as default values.